_dev/apps/ui/src/store/modules/product-feed/actions.ts

```diff
@@ -444,7 +444,7 @@ export default {
       issueLanguage: rootGetters[`app/${appGetters.GET_CURRENT_LANGUAGE}`],
       issueTimezone: encodeURI(Intl.DateTimeFormat().resolvedOptions().timeZone),
     });
-    const productFullId: string = `${payload.product.idProduct}${(+payload.product.idAttribute > 0 ? `-${payload.product.idAttribute}` : '')}`;
+    const productFullId: string = `${payload.product.idProduct}${(+payload.product.idAttribute > 0 ? `-${payload.product.idAttribute}` : '-0')}`;
```

Comment:
tu peux simplifier la ternaire du coup
```suggestion
    const productFullId: string = `${payload.product.idProduct}-${+payload.product.idAttribute > 0 ? payload.product.idAttribute : 0}`;
```